arcgis-ios-sdk-samples/Geometry/Format coordinates/FormatCoordinates.storyboard

```diff
@@ -178,16 +47,182 @@
+                                                    <rect key="frame" x="16" y="31.5" width="343" height="0.0"/>
+                                                    <nil key="textColor"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleBody"/>
+                                                    <textInputTraits key="textInputTraits" returnKeyType="done"/>
+                                                    <connections>
+                                                        <action selector="textFieldAction:" destination="geL-f0-eoD" eventType="editingDidEnd" id="R60-c0-Ieu"/>
+                                                        <outlet property="delegate" destination="geL-f0-eoD" id="QlL-Gw-ly6"/>
+                                                    </connections>
+                                                </textField>
+                                                <label opaque="NO" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" verticalCompressionResistancePriority="751" text="Decimal Degrees" textAlignment="natural" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" minimumScaleFactor="0.5" adjustsLetterSpacingToFitWidth="YES" adjustsFontForContentSizeCategory="YES" translatesAutoresizingMaskIntoConstraints="NO" id="5aq-8r-2lB">
+                                                    <rect key="frame" x="16" y="12" width="343" height="15.5"/>
+                                                    <fontDescription key="fontDescription" style="UICTFontTextStyleSubhead"/>
+                                                    <color key="textColor" white="0.33333333333333331" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
+                                                    <nil key="highlightedColor"/>
+                                                </label>
+                                            </subviews>
+                                            <constraints>
+                                                <constraint firstItem="5aq-8r-2lB" firstAttribute="leading" secondItem="297-yI-a6o" secondAttribute="leadingMargin" id="NRZ-fA-Vrf"/>
+                                                <constraint firstItem="Flo-zm-KFZ" firstAttribute="top" secondItem="5aq-8r-2lB" secondAttribute="bottom" constant="4" id="ija-gt-iDX"/>
+                                                <constraint firstItem="Flo-zm-KFZ" firstAttribute="trailing" secondItem="5aq-8r-2lB" secondAttribute="trailing" id="kVf-8F-EHK"/>
+                                                <constraint firstItem="Flo-zm-KFZ" firstAttribute="leading" secondItem="5aq-8r-2lB" secondAttribute="leading" id="qRW-it-UHG"/>
+                                                <constraint firstAttribute="trailingMargin" secondItem="5aq-8r-2lB" secondAttribute="trailing" id="vBM-bl-Cat"/>
```

Comment:
The trailing constraint of each label should be `greaterThanOrEqual` so that they are only as wide as they need to be.

Reply:
This doesn't affect the UI but okay.

Reply:
It doesn't affect the UI, but it does affect performance. Enable Color Blended Layers in the simulator to see how.

Before: 
![screen shot 2018-12-03 at 3 40 48 pm](https://user-images.githubusercontent.com/2257493/49408761-85bc9200-f712-11e8-85dd-d55ef28217e9.png)

After: 
![screen shot 2018-12-03 at 3 41 50 pm](https://user-images.githubusercontent.com/2257493/49408764-8bb27300-f712-11e8-81b5-0c96dab837f1.png)